features/prompt-registry/overview.mdx

```diff
@@ -188,6 +188,13 @@ promptlayer.templates.publish({
 
 </CodeGroup>
 
+### Composable Templates
```

Comment:
This should actually be it's own page in the Prompt Registry section.

Let's call them "Snippets", not composable templates

You should have multiple sections:
- Introduction (explaining why you would want to use them, explaining that they are automatically filled out in Playground, evals and when pulling it down in SDK)
- Referencing a snippet (explaining how to use it in your template)
- Using Labels and Versions (explaining how it is pulled at runtime, and why you would want to use a specific label or version)